README.md

```diff
@@ -43,3 +43,10 @@ Command                        | Description
 `npm run clean`                | Removes built artifacts from all sub-projects
 
 You should now be ready to work on any of the `<model-viewer>` projects!
+
+## Windows 10 Setup
+Due to dependency issues on Windows we recommend running `<model-viewer>` setup from a WSL2 environment.
+ * [WSL2 Install walkthrough](https://docs.microsoft.com/en-us/windows/wsl/install-win10)
+
+And installing Node.js via NVM.
```

Comment:
We should mention here the version, since that has been a problem for others. And maybe in the section above we should mention that we require npm 6.14, since 7 isn't working for us yet.

Reply:
Done.